service/worker/scanner/history/scavenger.go

```diff
@@ -333,15 +324,13 @@ func getTaskLoggingTags(err error, task taskDetail) []tag.Tag {
 			tag.WorkflowNamespaceID(task.namespaceID),
 			tag.WorkflowID(task.workflowID),
 			tag.WorkflowRunID(task.runID),
-			tag.WorkflowTreeID(task.treeID),
-			tag.WorkflowBranchID(task.branchID),
+			tag.WorkflowBranchToken(task.branchToken),
```

Comment:
I am not sure how useful this is without having it decoded. we also can't filter on this field basically...

Reply:
I merely followed the same pattern as `tag.ArchivalRequestBranchToken()`. If you're okay with dropping this, I can just remove it.

Reply:
Let's keep it. I was originally thinking maybe in error case we should decode the branch token and use the original treeID and branchID tag so during debugging we can filter on these tags. 
But second thought on this, we also only have branchToken in mutable state, not tree/branchID. So filtering on branchToken will actually make things easier.